packages/core/src/components/fields/SchemaField.js

```diff
@@ -206,7 +231,8 @@ function WrapIfAdditional(props) {
             <Label label={keyLabel} required={required} id={`${id}-key`} />
             <LabelInput
               label={label}
-              required={required}
```

Comment:
Why remove `required`?

Reply:
That seems to be needed. I just took `required` back, thank you!